tools/topology/topology2/cavs-sdw.conf

```diff
@@ -70,6 +70,11 @@ Define {
 	DEEP_BUFFER_PIPELINE_SRC        'mixin.15.1'
 	DEEP_BUFFER_PIPELINE_SINK       'mixout.1.1'
 	DEEP_BUFFER_PCM_NAME		'Deepbuffer Jack Out'
+	DEEP_BUFFER_PIPELINE_ID_2       16
+	DEEP_BUFFER_PCM_ID_2            35
+	DEEP_BUFFER_PIPELINE_SRC_2      'mixin.16.1'
+	DEEP_BUFFER_PIPELINE_SINK_2     'mixout.21.1'
+	DEEP_BUFFER_PCM_NAME_2          'Deepbuffer Amps'
```

Comment:
The PCM name shall be "Deepbuffer Speaker"

Reply:
and we have now double definition of this DB for Speaker and yes, I realize that the existing naming also uses "Deepbuffer Amps", which is incorrect, it should be really Speaker. We have Amp(lifier) on headset as well..